hist/hist/src/THStack.cxx

```diff
@@ -536,16 +536,16 @@ Double_t THStack::GetMaximum(Option_t *option)
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-///  returns the minimum of all added histograms
-///  returns the minimum of all histograms if option "nostack".
+///  Returns the minimum of all added histograms larger than minval.
+///  Returns the minimum of all histograms, larger than minval, if option "nostack".
 
-Double_t THStack::GetMinimum(Option_t *option)
+Double_t THStack::GetMinimum(Option_t *option, Double_t minval)
 {
    TString opt = option;
    opt.ToLower();
    Bool_t lerr = kFALSE;
    if (opt.Contains("e")) lerr = kTRUE;
-   Double_t them=0, themin = 1e300, c1, e1;
+   Double_t them = 0, themin = std::numeric_limits<float>::max(), c1, e1;
```

Comment:
Here it should be: 
std::numeric_Limits<double>::max() 
or (if you prefer using TMath)
TMath::Limits<double>::Max()